addon/copy.js

```diff
@@ -0,0 +1,97 @@
+import { assert } from '@ember/debug';
+import EmberObject from '@ember/object';
+import Copyable from './copyable';
+
+/**
+ @module ember-copy
```

Comment:
You shouldn't need to add `@module` anywhere with our version of YUIDoc, YUIDoc actually does supply filenames when you build plain JSON, so we infer the modules directly from that